canonical-web-and-design/juju.is
# Add the About overview page

## Done
- Updated the main navigation
- Add the about overview page

## QA
- Go to http://0.0.0.0:8041/about
- Check the main navigation matches the [menu copy doc](https://docs.google.com/document/d/1uh1HxU8lQ8pfNxP2dpPT0H6GbyQp3bkN2jv6NLrUhxI/edit)
- Check the page content matches the [copy doc](https://docs.google.com/document/d/1CSLdRlusY9BilYZ6CrmWeJblNEOWl62VfJh2J1yDfZk/edit#) 

Fixes https://github.com/canonical-web-and-design/juju.is/issues/94
